packages/lesswrong/lib/collections/revisions/newSchema.ts

```diff
@@ -486,6 +486,22 @@ const schema = {
       },
     },
   },
+  automatedContentEvaluations: {
+    graphql: {
+      outputType: "AutomatedContentEvaluation",
+      canRead: ["sunshineRegiment", "admins"],
+      resolver: async (revision, args, context) => {
+        // TODO: Do we need to do any access filtering here?
```

Comment:
@rjmk wasn't sure if we needed to check access here, or if setting `canRead` is sufficient.